plugins/hydra_ax_sweeper/example/banana.py

```diff
@@ -0,0 +1,24 @@
+# Copyright (c) Facebook, Inc. and its affiliates. All Rights Reserved
+import logging
+from typing import Any
+
+from omegaconf import DictConfig
+
+import hydra
+
+log = logging.getLogger(__name__)
+
+
+@hydra.main(config_path="conf/config.yaml")
+def banana(cfg: DictConfig) -> Any:
+    x = cfg.banana.x
+    y = cfg.banana.y
+    a = 1
+    b = 1
```

Comment:
Those are not the "standard" constants. I  think it's 1 and 100.
this might make the optimization significantly easier.

Reply:
Fixed